install.sh

```diff
@@ -1,12 +1,32 @@
 #!/bin/bash
 
-mkdir -p bin
+# shellcheck disable=SC2164
 
-cat src/*.sh > bin/temp.sh
-cat bashunit >> bin/temp.sh
-grep -v '^source' bin/temp.sh > bin/bashunit
-rm bin/temp.sh
+cd "$(dirname "$0")"
+rm -f lib/bashunit
+[ -d "lib" ] || mkdir lib
+cd lib
 
-chmod +x bin/bashunit
+TAG=${1-main}
 
-echo "Build complete. bashunit has been generated in the bin folder."
+if [[ $TAG == main ]]; then
+  echo "> Using main branch"
+  git clone git@github.com:TypedDevs/bashunit.git temp_bashunit
+  cd temp_bashunit
+  git pull
+  git checkout feat/install-and-build-scripts # temporal for testing
```

Comment:
This is just for testing this concept. If we are happy with it, and we want to merge it, we need to remove this line.